torch_xla/experimental/fori_loop.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import torch
+import torch_xla
+import torch_xla.core.xla_builder as xb
+import torch_xla.core.xla_model as xm
+import torch_xla.utils.utils as xu
+import torch_xla.core.xla_op_registry as xor
+
+from torch._C import DispatchKey
+from torch._ops import HigherOrderOperator
+import torch._higher_order_ops.while_loop
+from torch._higher_order_ops.while_loop import while_loop_op
+
+
+@while_loop_op.py_impl(DispatchKey.XLA)
+def while_loop(cond_fn, body_fn, operands):
+  # cond_fn&body_fn: callable
+  # operands: (Tuple of possibly nested dict/list/tuple of tensors)
+  return _xla_while_loop(cond_fn, body_fn, operands)
+
+
+def _xla_while_loop(cond_fn, body_fn, operands):
+
+  def op_fn(internal_x):
+    # TODO(manfei): replace cond_fn_placeholder and body_fn_placeholder after lower fn to xlacomputation and could be in xla::while
```

Comment:
nit. "# TODO(manfei) replace cond_fn_placeholder and body_fn_placeholder once xla::while lowering in the backend is available"

Reply:
Thanks, updated in the newest commit